### PR TITLE
feature:新增主机加解锁插件（need sync to 3.6.X)

### PR DESCRIPTION
**Pull request by Tanghaohaohao**
- 新功能
    - 新增CMDB-主机加锁与CMDB-主机解锁插件

### CODE REVIEW

File: pipeline_plugins/components/collections/sites/open/cc/host_lock/add_host_lock/v1_0.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+class CmdbAddHostLockService(AddHostLockMixin, CCHostLockBaseService):
+    pass
+
+
+class CmdbAddHostLockComponent(Component):
+    name = _("主机加锁")
+    code = "cmdb_add_host_lock"
+    bound_service = CmdbAddHostLockService
+    form = "%scomponents/atoms/cc/cmdb_add_host_lock.js" % settings.STATIC_URL
```
Comment (homholueng) [inline pipeline_plugins/components/collections/sites/open/cc/host_lock/add_host_lock/v1_0.py:40]: "%scomponents/atoms/cc/cmdb_add_host_lock/v1_0.js"

File: pipeline_plugins/components/static/components/atoms/cc/cmdb_delete_host_lock.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                hookable: true,
+                validation: [
+                    {
+                        type: "required"
+                    }
+                ]
+            }
+        },
+    ]
+})();
```
Comment (homholueng) [inline pipeline_plugins/components/static/components/atoms/cc/cmdb_delete_host_lock.js:56]: No newline at end of file

File: pipeline_plugins/components/static/components/atoms/cc/cmdb_add_host_lock.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                hookable: true,
+                validation: [
+                    {
+                        type: "required"
+                    }
+                ]
+            }
+        },
+    ]
+})();
```
Comment (homholueng) [inline pipeline_plugins/components/static/components/atoms/cc/cmdb_add_host_lock.js:56]: No newline at end of file

File: pipeline_plugins/components/collections/sites/open/cc/host_lock/delete_host_lock/v1_0.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+class CmdbDeleteHostLockService(DeleteHostLockMixin, CCHostLockBaseService):
+    pass
+
+
+class CmdbDeleteHostLockComponent(Component):
+    name = _("主机解锁")
+    code = "cmdb_delete_host_lock"
+    bound_service = CmdbDeleteHostLockService
+    form = "%scomponents/atoms/cc/cmdb_delete_host_lock.js" % settings.STATIC_URL
```
Comment (homholueng) [inline pipeline_plugins/components/collections/sites/open/cc/host_lock/delete_host_lock/v1_0.py:40]: "%scomponents/atoms/cc/cmdb_delete_host_lock/v1_0.js" % settings.STATIC_URL

File: pipeline_plugins/components/collections/sites/open/cc/host_lock/add_host_lock/v1_0.py (hunk 1 above)
Comment (Tanghaohaohao) [reply pipeline_plugins/components/collections/sites/open/cc/host_lock/add_host_lock/v1_0.py]: done

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-sops?pullRequest=2203) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-sops?pullRequest=2203) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-sops?pullRequest=2203) it.</sub>

File: pipeline_plugins/components/static/components/atoms/cc/cmdb_delete_host_lock.js (hunk 2 above)
Comment (Tanghaohaohao) [reply pipeline_plugins/components/static/components/atoms/cc/cmdb_delete_host_lock.js]: done

File: pipeline_plugins/components/static/components/atoms/cc/cmdb_add_host_lock.js (hunk 3 above)
Comment (Tanghaohaohao) [reply pipeline_plugins/components/static/components/atoms/cc/cmdb_add_host_lock.js]: done

File: pipeline_plugins/components/collections/sites/open/cc/host_lock/delete_host_lock/v1_0.py (hunk 4 above)
Comment (Tanghaohaohao) [reply pipeline_plugins/components/collections/sites/open/cc/host_lock/delete_host_lock/v1_0.py]: done

**Review by homholueng — APPROVED** [review]

**Pull request merged**